chain-core/src/tx/data/input.rs

```diff
@@ -13,7 +13,7 @@ use crate::tx::data::TxId;
 )]
 pub struct TxoPointer {
     pub id: TxId,
-    pub index: usize,
+    pub index: u64,
```

Comment:
here and `TxAccess`, perhaps add a TODO comment:
```
// TODO: u16 and Vec size check in Decode implementation
```

Reply:
done